UI/runScan.php

```diff
@@ -0,0 +1,100 @@
+<!-- 
+    When index.php is compiled and run on the terminal, it will launch this runScan.php script with messages visible.
+    Note: output messages built-into the crawler script are not visible on the server-side while running localhost:<port#> 
+    (i.e. during demonstration). However, at the end of the scan, output will be logged to the UI console available to view 
+    using: Fn F12.
+ -->
+<?php
+
+    // Allow capturing output buffer
+    ob_implicit_flush(true);
+    
+    // Locate and parse configuration file
+    // $config = parse_ini_file("./config.ini");
+    // $username = $config["username"];
```

Comment:
The username will be stored in a PHP `$_SESSION[<usr>]` variable. See: [link](https://www.w3schools.com/php/php_sessions.asp).